pkg/services/authz/rbac/service.go

```diff
@@ -336,6 +348,17 @@ func (s *Service) getAnonymousPermissions(ctx context.Context, ns claims.Namespa
 	return res.(map[string]bool), nil
 }
 
+// Renderer is granted permissions to read all dashboards and folders, and no other permissions
+func (s *Service) getRendererPermissions(ctx context.Context, action string) (map[string]bool, error) {
+	ctx, span := s.tracer.Start(ctx, "authz_direct_db.service.getRendererPermissions")
+	defer span.End()
+
+	if action == "dashboards:read" || action == "folders:read" {
```

Comment:
I'd have expected a need for `datasources:query` as well

Reply:
Ah, wait, I remember now - DS permission checks should not go through MT authZ service, right? That's why I didn't add them. But I'll put them in just in case.

Reply:
Ahhhhhh good point!

Reply:
is this related to: https://github.com/grafana/identity-access-team/issues/987#issue-2611131305